packages/modules/imodel-browser/src/containers/iModelGrid/useIModelData.ts

```diff
@@ -53,7 +55,11 @@ export const useIModelData = ({
     IModelSortOptions | undefined
   >(sortOptions && { ...sortOptions });
   const sortDescending = sortOptions?.descending;
-  const sortedIModels = useIModelSort(iModels, sortOptions);
+  const sortedIModels = useIModelSort(
+    iModels,
+    sortOptions,
+    requestType === "recents"
```

Comment:
Wouldn't you only sort if this is not recents?

Reply:
changed the logic of the function to instead provide the enableSort option instead of the disableSort to avoid this confusion